netcreateorg/netcreate-itest
# Fix Standalone Mode

Addressing Issue #68 

Resolution - `app/assets/index.html` is copied as-is to the `public` directory, but had not been updated to include the new ursys-core and ursys-addon libraries, which was the cause of the error.

### TEST

Follow the packaging instructions
- `npm run dev` once
- `npm run package` afterwards
- copy the contents of the `public` folder to your webserver docroot
- see if it works!
